datasets/glossis/glossis.py

```diff
@@ -60,38 +60,40 @@
         "prefix", type=str, nargs=1, help="Input folder/prefix", default="fews_glossis/"
     )
     parser.add_argument("assetfolder", type=str, nargs=1, help="GEE asset")
-
+    # 
+    parser.add_argument("gee_bucket_folder", type=str, nargs=1, help="GEE bucket folder")
     # TODO: change all these sections to separate commands and make sure they run independent
     # instead of creating one script to rule them all...
     parser.add_argument(
```

Comment:
I like this change! However, maybe good to note somewhere in the PR that everything now doesn't run by default?

Reply:
That was already the case because of the assetfolder argument